src/pages/home/home.ts

```diff
@@ -269,23 +273,26 @@ export class HomePage {
 
         //Adjust central location. Multiple instantaneous requests are ignored so just 1 is done every 1 second.
         userInfoSubject.debounce(() => Observable.timer(1000)).subscribe(() => {
+          this.isUserJoined = true;
           this.adjustCentralLocation({
             latitude: latitude,
             longitude: longitude,
           });
         });
 
         //Share observable among subscribers.
-        let userLeftObservable = this.groupSocketService.userLeftObservable.share();
+        const userLeftObservable = this.groupSocketService.userLeftObservable.share();
 
         //Remove pin of person who has left the group.
-        userLeftObservable.subscribe(userInfo => {
+        userLeftObservable.subscribe((userInfo: UserInfo) => {
```

Comment:
this.isUserJoined should be set to false here if there are no users in the space (not including you).